l10n_ch_lsv_dd/model/invoice.py

```
@@ -60,28 +58,25 @@ def cancel_payment_lines(self, cr, uid, ids, context=None):
             'mode': old_pay_order.mode.id,
         }
 
-        pay_order_id = pay_order_obj.create(cr, uid, vals, context)
+        pay_order_id = pay_order_obj.create(vals)
         wf_service = netsvc.LocalService('workflow')
-        wf_service.trg_validate(uid, 'payment.order', pay_order_id, 'cancel',
-                                cr)
-        pay_line_obj.write(cr, uid, pay_line_ids, {'order_id': pay_order_id},
-                           context)
+        wf_service.trg_validate('payment.order', pay_order_id, 'cancel')

```

Should raise an error. Method `wf_service.trg_validate` has not be ported (or I don't find the right solution) but it definition still has `uid` and `cr` as arguments